lib/normalize_headers_filter.js

```diff
@@ -0,0 +1,12 @@
+"use strict";
+
+const mwUtil = require('./mwUtil');
+
+module.exports = function normalizeHeaders(hyper, req, next) {
+    if (mwUtil.isNoCacheRequest(req)) {
+        req.headers['cache-control'] = 'no-cache';
+    } else {
```

Comment:
Nit: `else if (req.headers && req.headers['cache-control'])` (I wonder if this is needed, i.e. is there somewhere where we ensure the existence of the headers hash?)